spec/30-processing-model.md

```diff
@@ -0,0 +1,66 @@
+# Processing Model
+
+This section defines the behaviour of a tracing system when a new request is received by a component. As stated above tracing systems can decide for each individual request whether they want to _forward_ a trace or _participate_ in a trace. The processing model for both scenarios will be described individualy and adds a seperate processing model when a tracing system is the first system to start a trace. 
+
+## Processing Model in Forwarding Mode
+
+In forwarding mode a tracing system decides to forward tracing information but to not modify the trace-context headers. The request is then considered as not sampled. 
+
+1. Tracing system receives a `traceparent` header and may receive `tracecontext`. 
```

Comment:
I'd suggest to use something like "Tracing system or platform". The goal is that context propagation is implemented by platforms and tracing systems simply utilizing them.

Reply:
what is a platform in this context?